arangod/Cluster/ClusterInfo.cpp

```diff
@@ -547,20 +550,55 @@ DECLARE_HISTOGRAM(arangodb_load_current_runtime, ClusterInfoScale,
 DECLARE_HISTOGRAM(arangodb_load_plan_runtime, ClusterInfoScale,
                   "Plan loading runtimes [ms]");
 
+DECLARE_GAUGE(arangodb_internal_cluster_info_memory, std::size_t,
```

Comment:
I suggest to rename the metric so it is in line with many other memory usage metrics.
```suggestion
DECLARE_GAUGE(arangodb_internal_cluster_info_memory_usage, std::size_t,
```